app/controllers/items_controller.rb

```diff
@@ -259,21 +297,28 @@ def update_attributes
   #  ~ possibly 2 writes to WFS (each one triggers async object reindex),
   #  ~ plus an object save (more WFS requests, Fedora write, Solr reindex)
   def create_minimal_mods
-    unless Dor::WorkflowService.get_workflow_status('dor', @object.id, 'accessionWF', 'descriptive-metadata') == 'error' || Dor::WorkflowService.get_workflow_status('dor', @object.id, 'accessionWF', 'publish') == 'error'
-      render :text => 'Object is not in error for descMD or publish!', :status => 500
+    def accession_errors?(obj, task)
```

Comment:
Method definitions must not be nested. Use `lambda` instead.<br>Unused method argument - `obj`. If it's necessary, use `_` or `_obj` as an argument name to indicate that it won't be used.